### ISSUE TITLE
Not worked in JSX file

### ISSUE THREAD

**Issue by linonetwo** [FallenMax/styled-components-extractor#2]
When used in `.jsx` functional component:

```[SCE] Only `.jsx` and `.tsx` is supported``` showed up.

Example:

```jsx
// @flow
import React, { useEffect, useState } from 'react';
import styled from 'styled-components';

export default function Gallery() {
  return (
    <Container>
      <GalleryInfo>
        <TitleImage />
        <MetaData />
      </GalleryInfo>
      <GalleryContents>
        <Controls>
          <Pagination>
            <PaginationButton>{'<'}</PaginationButton>
            <PaginationButton>{'>'}</PaginationButton>
          </Pagination>
        </Controls>
        <ThumbNails>
          <ThumbNail />
        </ThumbNails>
        <Controls>
          <Pagination>
            <PaginationButton>{'<'}</PaginationButton>
            <PaginationButton>{'>'}</PaginationButton>
          </Pagination>
        </Controls>
      </GalleryContents>
      <Comments>
        <Comment>asdf</Comment>
      </Comments>
    </Container>
  );
}
```

Filename is `Gallery.jsx`.

If I change name to `Gallery.tsx` <- tsx here, then it works.

**Comment by FallenMax** [comment]
Hi, @linonetwo  
Sorry but I just tried and cannot reproduce on my machine, the extension can extract JSX tags from '.jsx' (note the notification toast)

![image](https://user-images.githubusercontent.com/2012170/62927389-06c34180-bde9-11e9-86d1-27ae9381ef88.png)

Do you have `JavaScript React` selected as language mode (see the red rectangle)? Can you reproduce with all extension disabled (use command `Reload Window With Extensions Disabled`)?

**Comment by linonetwo** [comment]
Ohch, My language is `babel javascript`, using https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel this extension. Can you support this language type also? It is widely used too.

I tried `Javascript React`, it actually works.

**Comment by linonetwo** [comment]
But looks it is just `javascriptreact` https://github.com/michaelgmcd/vscode-language-babel/blob/master/package.json#L98

I don't know it is what that made the difference.

**Comment by FallenMax** [comment]
@linonetwo Thanks, the problem is fixed in 0.0.5, upgrade the extension and it should work.

When `babel javascript` extension is enabled, `.jsx` is somehow recognized as `javascript`. I added support for `javascript` language mode and respect file extension (`.jsx`) properly.
